src/zenml/client.py

```diff
@@ -1160,6 +1160,8 @@ def update_stack(
 
         # Create the update model
         update_model = StackUpdate(  # type: ignore[call-arg]
+            workspace=self.active_workspace.id,
+            user=self.active_user.id,
             stack_spec_path=stack_spec_file,
```

Comment:
Sorry for this in advance, and I might be missing context to this bugfix, but instead of filling out useless fields here that are ignored in the backend. Why don't we remove these from Update models. Backwards compatibility should be preserved by having Extra.ignore on the base update model.

Reply:
Normally, that was the first version of the solution that I was going for. But later on, we ran into a hidden issue with the service connector updates. The issue was, that somewhere down the line, a service connector update model was converted to a request model and it failed due to the lack of these attributes. This is why I decided to revert the changes completely to a working state. We can think about how to handle the `update` models with the upcoming changes in the API.